scripts/tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "mode: count" > /tmp/coverage.out
+
+for d in $(go list ./... | grep -v vendor); do
```

Comment:
yea, I was mostly doing this since we aren't currently installing glide in the CI pipeline